docs/developer/available-plugins/user-emails.mdx

```diff
@@ -13,51 +13,77 @@ The payload is available directly from the email template.
 :::
 
 ## Events
-You can build template emails for below events:
-
-- Customer account has been created (_account_confirmation_)
-- Customer requested a reset password (_account_password_reset_)
-- Customer requested email change (_account_change_email_request_)
-- Customer confirmed email change (_account_change_email_confirm_)
-- Customer requested a delete account (_account_delete_)
-- Staff user has created a customer account (_account_set_customer_password_)
-- The invoice for the order is ready (_invoice_ready_)
-- The order has been created (_order_confirmation_)
-- The order has been confirmed by the staff user (_order_confirmed_)
-- The order has been fulfilled (_order_fulfillment_confirmation_)
-- The fulfillment has been updated (_order_fulfillment_update_)
-- The payment has been confirmed (_order_payment_confirmation_)
-- The order has been canceled (_order_canceled_)
-- The order has been refunded (_order_refund_confirmation_)
-
+[WebhookEventTypeEnum](developer/api-reference/enums/webhook-event-type-enum.mdx) contain all availabe events.
 
 ## Templates
 
 `UserEmails` plugin uses [handlebars](https://handlebarsjs.com/guide/) as a template language. 
 You can use [Saleor's default templates](https://github.com/saleor/saleor-email-templates) as a good start for building your emails. 
 Prepared emails can be uploaded via plugin section on the Dashboard side. 
 
-
 ## Helper functions
-The plugin offers below helper fuction:
 
-- `format_address(address, include_phone=True, inline=False, latin=False)` - converts json address structure to string.
+### format_address
+Applies formatting to the address.
+
+Arguments:
+- **address** (json) - Adress instance in json format.
+- **include_phone** (boolean) - Controls if formatted adress will contain phone number. Default: True
+- **inline** (boolean) - Controlls if address should be formated inline. Default: False
+- **latin** (boolean) - ???
```

Comment:
It is a value passed to i18naddress library. 
https://github.com/mirumee/google-i18n-address#address-latinization